deepracer/logs/log_utils.py

```diff
@@ -454,6 +454,14 @@ def plot(ax, df, xval, xlabel, yval, ylabel, title=None):
         ax.set_ylabel(ylabel)
         ax.set_xlabel(xlabel)
 
+        # Draw the running average
```

Comment:
This should be optional. (So to include a variable in the method to turn it on or off.)